lemur/default.conf.py

```diff
@@ -16,3 +16,13 @@
 
 LOG_LEVEL = "DEBUG"
 LOG_FILE = "lemur.log"
+
+# Set of controls to use around ingesting user group information from the IDP
```

Comment:
It would also be great to document these new configurations in the Lemur docs for better visibility and understanding. Thanks.